CONTRIBUTING.md

```diff
@@ -348,13 +348,32 @@ description: {
 }
 ```
 
+Changelogs may also be written in "markdown-frontmatter" format. This is useful if your
+description contains braces, which must be escaped with backslashes in `.cabal` file
+format.
+
+```markdown
+---
+synopsis: Add feature xyz
+packages: [cabal-install]
+prs: #0000
+issues: #0000 #0000
```

Comment:
Oh, this needs a followup: in YAML, `#` is a comment marker, so the changelog-d YAML format doesn't use the `#` in issue/PR numbers.

Reply:
I copied that from `changelog-d`'s `README`; I guess that was what [this](https://github.com/haskell/cabal/pull/10383#issuecomment-2378858091) was talking about.